pkg/framework/framework.go

```diff
@@ -55,11 +55,24 @@ func NewFramework(userName string, stageConfig ...utils.Options) (*Framework, er
 	return NewFrameworkWithTimeout(userName, time.Second*60, stageConfig...)
 }
 
-func NewFrameworkWithTimeout(userName string, timeout time.Duration, options ...utils.Options) (*Framework, error) {
+// This periodically refreshes framework for Stage user because of Keycloak access token expires in 15 minutes
+func refreshFrameworkStage(currentFramework *Framework, userName string, timeout time.Duration, options ...utils.Options) {
+	for {
+		time.Sleep(time.Minute * 10)
```

Comment:
Why hardcode to sleep 10 minutes? From my experience, it may expire sooner.

Reply:
Is it possible to use a param to pass the sleep time in?

Reply:
@psturc , do you have any suggestion here since you also use staging cluster in your test?

Reply:
I actually don't have much experience with testing against stage cluster, but given the fact that Jan has tested it locally and it worked, I assume it could be generally used for all tests that are running against staging.

But just to be super careful - would it make sense to you @jinqi7 to test Jan's PR locally with release-pipelines-staging tests before merging?

Reply:
I will test to refresh FW every 10 minutes in release test manually. But the test result may be different when it's running in the nightly test or PR's test.

Reply:
It's not work for me with refreshing it every10 minutes.

Reply:
I just used a ticker to refresh FW in my tests - such as:  [link](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/release/pipelines/rh_advisories.go).

Reply:
hmm that's strange... I cloned Jan's PR, updated [rhtap-demo test](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) to run the test for more than 15 minutes, ran it against stage cluster (in my workspace) and it passed...
```
❯ ginkgo build ./cmd && ginkgo -p -v --label-filter='verify-stage' ./cmd/cmd.test
...
Ran 6 of 414 Specs in 1057.116 seconds
SUCCESS! -- 6 Passed | 0 Failed | 31 Pending | 377 Skipped


Ginkgo ran 1 suite in 17m40.623228849s
Test Suite Passed
```

Reply:
Maybe I keeps using my current way and you go ahead.  I may take more test with it later.